SmartDeviceLink/private/SDLVoiceCommandManager.m

```diff
@@ -137,7 +137,12 @@ - (void)setVoiceCommands:(NSArray<SDLVoiceCommand *> *)voiceCommands {
         [weakSelf sdl_updatePendingOperationsWithNewCurrentVoiceCommands:newCurrentVoiceCommands];
     }];
 
-    [self.transactionQueue cancelAllOperations];
+    for (NSOperation *operation in self.transactionQueue.operations) {
+        if (operation.isExecuting && !operation.isFinished && !operation.isCancelled) {
+            [operation cancel];
+            break;
+        }
+    }
```

Comment:
This seems incorrect. You have changed cancelling all operations to only change the executing one. So if you have operations:

```
Op1 - executing
Op2 - waiting
```

And add Op3, this code used to:

```
Op1 - canceled
Op2 - canceled
Op3 - waiting
```

The next op to execute would be op 3, correctly. With your change:

```
Op1 - cancelled
Op2 - waiting
Op3- waiting
```

As you can see, this is incorrect.